presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestDoubleMaxAggregation.java

```diff
@@ -16,15 +16,50 @@
 import com.facebook.presto.common.block.Block;
 import com.facebook.presto.common.block.BlockBuilder;
 import com.facebook.presto.common.type.StandardTypes;
+import com.facebook.presto.metadata.FunctionAndTypeManager;
+import com.facebook.presto.metadata.MetadataManager;
+import com.facebook.presto.spi.function.JavaAggregationFunctionImplementation;
 import com.google.common.collect.ImmutableList;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
 
 import java.util.List;
 
+import static com.facebook.presto.block.BlockAssertions.createDoublesBlock;
 import static com.facebook.presto.common.type.DoubleType.DOUBLE;
+import static com.facebook.presto.operator.aggregation.AggregationTestUtils.assertAggregation;
+import static com.facebook.presto.sql.analyzer.TypeSignatureProvider.fromTypes;
+import static java.lang.Double.NEGATIVE_INFINITY;
+import static java.lang.Double.NaN;
+import static java.lang.Double.POSITIVE_INFINITY;
 
 public class TestDoubleMaxAggregation
         extends AbstractTestAggregationFunction
 {
+    private static final String FUNCTION_NAME = "max";
+    private JavaAggregationFunctionImplementation maxFunction;
+
+    @BeforeClass
+    public void setup()
+    {
+        FunctionAndTypeManager functionAndTypeManager = MetadataManager.createTestMetadataManager().getFunctionAndTypeManager();
```

Comment:
AbstractTestAggregationFunction (which this extends) already has a function and type manager and a method getFunction() for getting the function implementation using that. Can we use that and just inline the call to getFunction() in the test instead of adding a setup?